pkgs/development/ocaml-modules/ppx_deriving_yojson/default.nix

```diff
@@ -4,15 +4,15 @@
 
 buildDunePackage rec {
   pname = "ppx_deriving_yojson";
-  version = "3.5.1";
+  version = "3.5.2";
 
   minimumOCamlVersion = "4.04";
 
   src = fetchFromGitHub {
     owner = "ocaml-ppx";
     repo = "ppx_deriving_yojson";
     rev = "v${version}";
-    sha256 = "13nscby635vab9jf5pl1wgmdmqw192nf2r26m3gr01hp3bpn38zh";
```

Comment:
This hash seems wrong:

> hash mismatch in fixed-output derivation '/nix/store/rcws8q6q6k84gs0vxsvwkk2x6xpkiy5c-source':
  wanted: sha256:098rpjbykp7ffhs62mgxlk7349l665xh1w1m8ldj6rjb690cc945
  got:    sha256:1vbhmnhnj1aa4jrp8xqi52nggwj7vrml83z2j0r0qzvl65v02mc0

Reply:
Indeed -- not sure if my mistake or if it changed under me for some reason. Anyways, fixed now.